### PR TITLE
OBSDOCS-1519 - Power monitoring Tech Preview 0.3 release notes

### PR DESCRIPTION
**Pull request by shreyasiddhartha**
Change type: Doc update; Power Monitoring - Tech Preview Release Notes 0.3
Doc JIRA: https://issues.redhat.com/browse/OBSDOCS-1519

Fix Version: 4.14+

Doc Preview: https://85107--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/power_monitoring/power-monitoring-release-notes.html#power-monitoring-release-notes-0-3

NOTE TO THE REVIEWER: Removed the known issue section, as it is no longer applicable, on QE's request.

SME Review: @sthaha 
QE Review: @vprashar2929 
Peer Review: @bergerhoffer

### CODE REVIEW

**Review by vprashar2929 — APPROVED** [review]
/lgtm

**Review by sthaha — COMMENTED** [review]
/lgtm

**Review by sthaha — APPROVED** [review]

File: observability/power_monitoring/power-monitoring-release-notes.adoc
Comment (vprashar2929) [inline observability/power_monitoring/power-monitoring-release-notes.adoc:33]: @shreyasiddhartha Do we add link to bug id for tracking in release notes?

**Review by vprashar2929 — APPROVED** [review]
/lgtm

File: observability/power_monitoring/power-monitoring-release-notes.adoc
Comment (shreyasiddhartha) [reply observability/power_monitoring/power-monitoring-release-notes.adoc:33]: @vprashar2929 Tried doing that but my PR failed saying "Couldn't include links that requires login". So I am not adding links for now.

**Review by bergerhoffer — COMMENTED** [review]
A few suggestions!

File: observability/power_monitoring/power-monitoring-release-notes.adoc
Comment (shreyasiddhartha) [reply observability/power_monitoring/power-monitoring-release-notes.adoc:28]: Yes that's right. It is supposed to go live on the release date. I haven't seen any observability component listing the release date. Since Power Monitoring comes under observability, I will keep it as it is to maintain consistency.

File: observability/power_monitoring/power-monitoring-release-notes.adoc
Comment (bergerhoffer) [reply observability/power_monitoring/power-monitoring-release-notes.adoc]: Interesting. I think the release notes headings should always be plural (like we say that "Prerequisites" should always be plural even if it's just one). But I'll bring it up to my peer review squad to see what others think!

Comment (shreyasiddhartha) [reply observability/power_monitoring/power-monitoring-release-notes.adoc]: I can change it if the merge reviewer asks me to. :) I am adding the tag as this needs to go out today.

File: observability/power_monitoring/power-monitoring-release-notes.adoc
Comment (xenolinux) [inline observability/power_monitoring/power-monitoring-release-notes.adoc:31]:
```suggestion
=== Bug fixes
```

**Review by xenolinux — COMMENTED** [review]
One suggestion to use plural. Once that's fixed; PR can be merged.

File: observability/power_monitoring/power-monitoring-release-notes.adoc
Comment (shreyasiddhartha) [reply observability/power_monitoring/power-monitoring-release-notes.adoc]: @xenolinux Done! :) Thank you.

Comment (xenolinux) [reply observability/power_monitoring/power-monitoring-release-notes.adoc]: Thanks. I will merge the PR once builds pass.

**Pull request merged**
